minimesos/src/main/java/com/containersol/minimesos/cluster/MesosCluster.java

```diff
@@ -52,6 +52,9 @@
 
     /**
      * Create a new MesosCluster with a specified cluster architecture.
+     *
+     * @param clusterConfig TODO(mwl)
+     * @param processes TODO(mwl)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://ci.infra.container-solutions.com/sonar/coding_rules#rule_key=squid%3AS1135)